models/base_sources/goerli_base_sources.yml

```diff
@@ -0,0 +1,293 @@
+version: 2
+
+sources:
+  # Base Tables
+  - name: goerli
+    description: "Goerli raw tables including transactions, traces and logs."
+    freshness:
+      warn_after: { count: 12, period: hour }
+    tables:
+      - name: transactions
+        loaded_at_field: block_time
+        description: "An Goerli transaction refers to an action initiated by an externally-owned account (i.e., an account managed by a human, not a contract)."
+        columns:
+          - &block_date
+            name: block_date
+            description: "Block event date in UTC"
+          - &block_time
+            name: block_time
+            description: "Timestamp for block event time in UTC"
+          - &value
+            name: value
+            description: "Amount of ETH transferred from sender to recipient"
+          - &block_number
+            name: block_number
+            description: "Block number"
+          - name: gas_limit
+            description: "Maximum amount of gas units that can be consumed by the transaction"
+          - name: gas_price
+            description: "Gas price denoted in gwei, which itself is a denomination of ETH - each gwei is equal to 10-9 ETH"
+          - name: gas_used
+            description: "Number of gas units consumed by the transaction"
+          - name: max_fee_per_gas
+            description: "Maximum amount of gas willing to be paid for the transaction"
+          - name: max_priority_fee_per_gas
+            description: "Maximum amount of gas to be included as a tip to the miner"
+          - name: base_fee_per_gas
+            description: "Market price for gas"
+          - name: nonce
+            description: "Number of confirmed transactions previously sent by this account"
+          - name: index
+            description: "Transaction index"
+          - &success
+            name: success
+            description: "Whether the transaction was completed successfully"
+          - &from
+            name: from
+            description: "Wallet address that initiated the transaction"
+          - &to
+            name: to
+            description: "Wallet address that received the transaction"
+          - &block_hash
+            name: block_hash
+            description: "Primary key of the block"
+          - name: data
+            description: "Any binary data payload"
+          - name: hash
+            description: "Primary key of the transaction"
+            tests:
+              - not_null
+          - name: type
+            description: "Transaction type"
+          - name: access_list
+            description: "Specifies a list of addresses and storage keys"
+            description: "Specifies a list of addresses and storage keys"
+          - name: l1_block_number
```

Comment:
@hildobby I think there are a lot of columns that were copied from the Optimism transactions table but that don't exist on Fantom here

Reply:
🫠 alright, let me fix this real quick